app/src/main/java/com/codesoom/assignment/application/ProductDeleteServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+
+@RequiredArgsConstructor
+@Service
+public class ProductDeleteServiceImpl implements ProductDeleteService {
+
+    private final ProductRepository repository;
+
+    @Transactional
+    @Override
+    public void deleteById(Long id) {
+        final Product product = repository.findById(id).orElseThrow(() -> new ProductNotFoundException(id));
```

Comment:
한 줄이 너무 깁니다. 한 줄에 점이 하나 오도록 적절히 개행해 주세요.
```suggestion
        final Product product = repository.findById(id)
            .orElseThrow(() -> new ProductNotFoundException(id));
```